packages/query/src/stacks/bns/bns.hooks.ts

```diff
@@ -0,0 +1,7 @@
+import { useGetBnsNamesOwnedByAddressQuery } from './bns.query';
+
+export function useCurrentAccountNames(address: string) {
+  return useGetBnsNamesOwnedByAddressQuery(address, {
+    select: resp => resp.names ?? [],
+  });
+}
```

Comment:
Not needed here, but wonder if we should just remove entirely, as what's the real value? it saves us having to access a nested prop from useGetBnsNamesOwnedByAddressQuery?